apps/accounts/forms.py

```diff
@@ -33,23 +31,25 @@ class SignupForm(UserCreationForm):
                                 label=_("Password"))
     password2 = forms.CharField(widget=forms.PasswordInput,
                                 max_length=120,
-                                label=_("Password (again)"),
-                                help_text=_("We are asking you to re-enter "
-                                            "your chosen password to make "
-                                            "sure it was entered correctly."))
+                                label=_("Password (again)"))
+    identification_choice = forms.ChoiceField(label="Your Role", choices=[])
```

Comment:
can this be done directly. Possibly with a [ModelChoiceField](https://docs.djangoproject.com/en/2.2/ref/forms/fields/#django.forms.ModelChoiceField) or a [MultipleModelChoiceField](https://docs.djangoproject.com/en/2.2/ref/forms/fields/#modelmultiplechoicefield) with a different widget?

Reply:
TY! This is nicer. I changed it to use ModelChoiceField .